models/configuration.rb

```diff
@@ -79,11 +79,13 @@ class Restitution < Base
     field :geometry, default: []
     field :intermediate_solutions, default: true
     field :csv, default: false
+    field :use_deprecated_csv_headers, default: false
     field :allow_empty_result, default: false
   end
 
   class Schedule < Base
     field :range_indices, default: nil # extends schedule_range_date
+    field :range_date, default: nil # should only be used to return consistent date in result. TODO: move to result structure
```

Comment:
To avoid any misuse of this parameter, could we only store the starting date ?

Reply:
Good idea !